contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -2324,10 +2324,19 @@ modify_insert_stmt(InsertStmt *stmt, Oid relid)
 	HeapTuple	tuple;
 	List	   *insert_col_list = NIL,
 			   *temp_col_list;
+	char		relkind = get_rel_relkind(relid);
 
 	if (!output_into_insert_transformation)
```

Comment:
Issue (blocking): Hmm, this is not going to work for OUTPUT INTO clause in a INSERT statement as `output_into_insert_transformation` is going to be true in that case. We need to have a proper conditional to verify if this the `insert statement` of a OUTPUT INTO clause we are dealing with here.